src/app/georeferencer/qgsgcplist.cpp

```diff
@@ -30,16 +33,27 @@ QgsGCPList::QgsGCPList( const QgsGCPList &list )
   }
 }
 
-void QgsGCPList::createGCPVectors( QVector<QgsPointXY> &mapCoords, QVector<QgsPointXY> &pixelCoords )
+void QgsGCPList::createGCPVectors( QVector<QgsPointXY> &mapCoords, QVector<QgsPointXY> &pixelCoords, const QString &wktProj )
 {
+  QgsCoordinateReferenceSystem projTarget( wktProj );
+  QgsCoordinateTransform transformer;
```

Comment:
Not used.

Reply:
Unsure if its a good idea to keep the transform when projections haven't changed or if better to make a new one each time.